<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matiere</class>
 <widget class="QWidget" name="matiere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="matiere_menu">
    <widget class="QPushButton" name="menu">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>490</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Main Menu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Update subject</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>671</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 22pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string>                  subject Managment </string>
     </property>
    </widget>
    <widget class="QPushButton" name="show">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>400</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Show subject</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteS">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>320</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete subject</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Adder">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Add subject</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="add_matiere">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>490</y>
       <width>191</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>601</width>
       <height>391</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="ID">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>     Enter ID Subject</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Name">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2_Name">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>      Enter Name of The Subject</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coefficient">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3_coef">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>   Enter  Coefficent of The Subject</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4_professor">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Select  the professor</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cnrps">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>300</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>                     CNRPS</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="show_matiere">
    <widget class="QLineEdit" name="searchField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search For Student</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>-20</y>
       <width>511</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 22pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>                Liste of Students </string>
     </property>
    </widget>
    <widget class="QTableView" name="mat_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>891</width>
       <height>381</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Goback">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>510</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="update_professor">
    <widget class="QPushButton" name="updateC">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>310</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>update</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_up">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="NewProf">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>350</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter new professor :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateN">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>240</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>update</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter the ID of the Sbject</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Name_B">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>240</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateP">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>380</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_U">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="coef_P">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>310</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="coef_B">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>coef</string>
     </property>
    </widget>
    <widget class="QLabel" name="newCoef">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>280</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter new coefficent:</string>
     </property>
    </widget>
    <widget class="QLabel" name="newName">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>enter new Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Name_P">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>240</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="prof_B">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>380</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>professor</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prof_P">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>380</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="table_M_update">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>261</width>
       <height>571</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="notificationLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>170</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="GoB">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Go back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="delete_matiere">
    <widget class="QLineEdit" name="coef">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>280</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>220</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>400</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushbotton{ }
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>DELETE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cnrps_mat">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>340</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="table_del">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>591</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="ID_mat">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>160</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="go_b_del">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Go back</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
